easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.3.0-intel-2014b.eb

```diff
@@ -0,0 +1,46 @@
+name = 'OpenFOAM'
+version = '2.3.0'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package. 
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer, 
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+source_urls = ['http://downloads.sourceforge.net/foam/%(version)s']
+sources = [
+    SOURCE_TGZ,
+    'ThirdParty-%(version)s.tgz',
+]
+
+paraversion = '4.2.0'
+
+patches = [
+    'OpenFOAM-%(version)s_bugfix1.patch',
+    'cleanup-OpenFOAM-%(version)s.patch',
+    'OpenFOAM-%(version)s_libreadline.patch',
+    ('cleanup-ThirdParty-%(version)s.patch', ".."),  # patch should not be applied in OpenFOAM subdir
+    'external-3rd-OpenFOAM-%(version)s.patch',
+    ('external-3rd-ThirdParty-%(version)s.patch', ".."),
+    'external_paraview_%s.patch' % paraversion,
+]
+
+dependencies = [
+    ('CGAL', '4.5', '-Python-2.7.8'),
+    ('libreadline', '6.3'),
+    ('METIS', '5.1.0', '-32bitIDX'),
```

Comment:
I do not understand this comment. Sould I use a versionsuffix for OpenFOAM? The default index for METIS is 32 bit (as it compiles in the version included in OpenFOAM), but all our versions use a patch to make the index 64 bit.